### PR TITLE
Optimization for one row inference in DF

### PR DESCRIPTION
**Pull request by ShvetsKS**
# Description
Optimization for one row inference in DF classification

### CODE REVIEW

**Comment by ShvetsKS** [comment]
make test

**Comment by ShvetsKS** [comment]
make test

File: algorithms/kernel/dtrees/forest/classification/df_classification_predict_dense_default_batch.h [hunk 1]
```diff
@@ -57,8 +61,14 @@ class PredictKernel : public daal::algorithms::Kernel
      *  \param r[out]   Prediction results
      *  \param par[in]  decision forest algorithm parameters
      */
+    PredictKernel(): _task(nullptr){};
+    ~PredictKernel()
+    {
+        if(!_task)    delete _task;
```
Comment (vepifanov) [inline algorithms/kernel/dtrees/forest/classification/df_classification_predict_dense_default_batch.h]: Typo: if (!_task) should be if (_task).

Comment (ShvetsKS) [reply algorithms/kernel/dtrees/forest/classification/df_classification_predict_dense_default_batch.h]: fixed, thanks

**Comment by ShvetsKS** [comment]
make test

**Comment by ShvetsKS** [comment]
issue: https://github.com/intel/daal/issues/271
was fixed in this commit.

**Comment by ShvetsKS** [comment]
PCm:  http://intel-ci.intel.com/ea410be6-f3bb-f1a9-9995-8cdcd4b723a1

**Pull request merged**